project-icp/bee-pollinator-app
# Update Land Cover Mods and Report Cover Crops With Main Crop

# Overview

- Replaces placeholder land cover mods with a more finalized list (bye, bye buckwheat 🌱 You will be missed )
- Modifies the model to aggregate crops over the updated land cover mods and each of their cover crop types (model only sends back results under the main crop's id)

Connects #119

#### Notes
Updated `cropTypes.json` to fix the chart titles. The legend consumes it, too. We may want to filter it or something to get rid of the cover crops which won't appear in the raster:
![screen shot 2017-02-01 at 4 28 33 pm](https://cloud.githubusercontent.com/assets/7633670/22528579/edc918a4-e8a1-11e6-8bb4-8dfef0d4a002.png)

Left updating the colors for a time when things are more settled
![screen shot 2017-02-01 at 4 28 37 pm](https://cloud.githubusercontent.com/assets/7633670/22528582/f054b498-e8a1-11e6-8b63-b2e967238321.png)


# Demo

![screen shot 2017-02-01 at 5 03 00 pm](https://cloud.githubusercontent.com/assets/7633670/22528556/dc88e98e-e8a1-11e6-9327-33a510025235.png)

Just copy and pasted whatever colors for testing purposes:
![screen shot 2017-02-01 at 5 14 22 pm](https://cloud.githubusercontent.com/assets/7633670/22528572/e7952810-e8a1-11e6-9cce-60bf738567d4.png)


# Testing instructions

* pull branch 
* reinstall python model `vagrant ssh && sudo python /vagrant/src/icp/pollinator/setup.py install`
* rebundle
* Draw an area of interest in the app and confirm all crops and their crop types get grouped together in the charts
